core/types/receipt.go

```diff
@@ -43,6 +43,9 @@ const (
 
 	// ReceiptStatusSuccessful is the status code of a transaction if execution succeeded.
 	ReceiptStatusSuccessful = uint64(1)
+
+	// DefaultEffectiveGasPrice is the default effective gas price for a transaction.
+	DefaultEffectiveGasPrice = 100
```

Comment:
Hey @Frozen, I believe @sophoah mean 100GWEI which is `100000000000` as int or `0x174876e800` as hex

Proof:
```
$ curl -d '{
  "jsonrpc":"2.0",
  "method":"hmy_gasPrice",
  "params":[],
  "id":1
}' -H 'Content-Type:application/json' -X POST 'https://api.s0.b.hmny.io'
```
Response: `{"jsonrpc":"2.0","id":1,"result":"0x174876e800"}`

And
```
$ curl -d '{
  "jsonrpc":"2.0",
  "method":"hmyv2_gasPrice",
  "params":[],
  "id":1
}' -H 'Content-Type:application/json' -X POST 'https://api.s0.b.hmny.io'
```
Response `{"jsonrpc":"2.0","id":1,"result":100000000000}`

Reply:
What I have right now is `100` as v2 and `0x64` as hex for v1

Reply:
updated to 100 gwei